shapes/profile/app.shex

```diff
@@ -0,0 +1,36 @@
+PREFIX oidc: <http://www.w3.org/ns/solid/oidc#>
+PREFIX foaf: <http://xmlns.com/foaf/0.1/>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX schem: <http://schema.org/>
+PREFIX vcard: <http://www.w3.org/2006/vcard/ns#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX acl: <http://www.w3.org/ns/auth/acl#>
+PREFIX cert:  <http://www.w3.org/ns/auth/cert#>
+PREFIX ldp: <http://www.w3.org/ns/ldp#>
+PREFIX sp: <http://www.w3.org/ns/pim/space#>
+PREFIX solid: <http://www.w3.org/ns/solid/terms#>
+
+<#id> {
+  a schem:SoftwareApplication
+    // rdfs:comment  "A software application. (from Schema.org)" ;
+  oidc:redirect_uris IRI + 
+    // rdfs:comment  "URIs where the Solid Identity Provider may redirect the user to complete the login process" ;
+  oidc:scope xsd:string ?
+    // rdfs:comment  "A string containing a space-delimited list of OAuth2.0 scopes your application is allowed to request." ;
```

Comment:
We need general guidance on whether or not to end descriptive message strings with full stops or not (this file sometimes ends with full stops (like this line), but others don't).
My recommendation is that intentionally short descriptive strings (e.g., `rdfs:label`) don't end in full stops, but intentionally longer descriptive strings (e.g., `rdfs:comment`, or `shacl:message`, etc.) do have full stops (since they can often be multiple sentences).
(I've added that recommendation to the still open PR on vocab guidance here: https://github.com/inrupt/public-documentation/pull/10/files, and it can be discussed there: https://github.com/inrupt/public-documentation/pull/10/commits/80f793f6f4651fdd794c5f27c9c2a3c62d25db69)